tests/zeek/close-connection.spicy

```diff
@@ -0,0 +1,30 @@
+# @TEST-EXEC: spicyz -o test.hlto %INPUT test.evt
+# @TEST-EXEC: ${ZEEK} -b -r ${TRACES}/long-dns-connection.pcap _Zeek::Spicy test.hlto
```

Comment:
This is not actually checking that `close_connection()` indeed has the intended effect. I think this should generate multiple entries in `conn.log` now instead of just one as it would have previously done, right? So could do a `btest-diff` on `conn.log` to confirm that.